app/views/pages/home.html.erb

```diff
@@ -8,74 +8,45 @@
+              alt="Placeholder image"
+            />
+          </figure>
+        </div>
       </div>
-    </div>
-    <div class="column is-8">
-      <div class="card-content p-0">
-        <div class="media">
-          <div class="media-content">
-            <p class="title is-4 m-0">Yard Sale!</p>
-            <time class="p-0" datetime="2016-1-1"> 1 Jan 2016, 07:00am</time>
+      <div class="column is-8">
+        <div class="card-content p-0">
+          <div class="media">
+            <div class="media-content">
+              <p class="title is-4 m-0"><%= yardsale.title %></p>
+              <time class="p-0" datetime="2016-1-1">
+                <%= yardsale.date.to_date.strftime("%A, %B %d, %Y") %></time
+              >
+            </div>
           </div>
-        </div>
-        <div class="content">
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus nec
-          iaculis mauris.
-          <br />
-          <br />
-          <div>
-            <span class="tag is-primary">Collectibles</span>
-            <span class="tag is-primary">Clothing</span>
-            <span class="tag is-primary">Electronics</span>
+          <div class="content">
+            <%= truncate(yardsale.description, length:200) %>
+            <br />
+            <br />
+            <div>
+              <span class="tag is-primary">Collectibles</span>
+              <span class="tag is-primary">Clothing</span>
+              <span class="tag is-primary">Electronics</span>
```

Comment:
Is this hardcode for testing purposes?

Reply:
Yes, i will include tags in the seed file

Reply:
tags included in the seedfile. Now mapping though the tags